Source/Response.swift

```diff
@@ -154,6 +183,8 @@ public struct DownloadResponse<Value> {
     /// The timeline of the complete lifecycle of the request.
     public let timeline: Timeline
 
+    var _metrics: AnyObject?
```

Comment:
Yep...you can't store a `URLSessionTaskMetrics` as a stored property b/c it's not available on the older OS's. This is the only valid workaround I've found to the problem yet. It's nasty, but it does work.